lib/net.c

```diff
@@ -1108,6 +1123,14 @@ start_iface(struct gatekeeper_if *iface)
 		goto stop_partial;
 
 out:
+	ret = rte_eth_dev_set_mtu(iface->id, iface->mtu);
+	if (ret < 0) {
+		RTE_LOG(ERR, PORT,
+			"net: cannot set the MTU on the %s iface\n",
+			iface->name);
+		goto stop_partial;
+	}
+
 	iface->hw_filter_eth = rte_eth_dev_filter_supported(iface->id,
 		RTE_ETH_FILTER_ETHERTYPE) == 0;
 	RTE_LOG(NOTICE, PORT,
```

Comment:
It's not related to the patch itself and I cannot comment on the `if` statement just after label `ipv4_acls`: `if (!iface->hw_filter_ntuple && ipv4_if_configured(iface))`. But, shouldn't that test be replaced with `ipv4_acl_enabled(iface)`?